joinservice/internal/server/server.go

```diff
@@ -114,6 +120,20 @@ func (s *Server) IssueJoinTicket(ctx context.Context, req *joinproto.IssueJoinTi
 		return nil, status.Errorf(codes.Internal, "generating ssh emergency CA key: %s", err)
 	}
 
+	principalList := req.HostCertificatePrincipals
+	additionalPrincipals, err := s.fileHandler.Read(constants.SSHAdditionalPrincipalsPath)
```

Comment:
This file is only written by the initserver right now. If we want to use it on all control plane nodes, we need to write it in the joinclient, too.